cmd/xmidt-agent/config.go

```diff
@@ -258,6 +258,7 @@ var defaultConfig = Config{
 		},
 	},
 	Websocket: Websocket{
+		Enabled:               true,
 		URLPath:               "api/v2/device",
```

Comment:
added a new default to address my comment
https://github.com/xmidt-org/xmidt-agent/pull/60#pullrequestreview-1980095677

Reply:
this is fine but does disabled make more sense because default bool is false? I missed this pattern so I can switch the mocktr181 to use disabled if that's better

Reply:
because of the power of goschtalt, either one is fine or works for me

Reply:
Hmm - the question I have is: 
If this defaults to on, will the default configuration actually work for any service?

If you need to provide some configuration to make ti work, I think defaulting it to off is ok.  If it will work just fine without any non-default values, then I agree that `Disabled` would make more sense.

Reply:
what do you think @johnabass ?

Reply:
@denopink and I chatted & while well intended, I'm not merging this PR for a few reasons:

1. When we enable QUIC as a protocol, we'll know better how we want to refactor the config & yes, it will be a breaking change, but that's probably better than guessing about several things and being wrong & then still have a breaking change later.
2. This doesn't help us reach our initial goal/deadline.

Reply:
Thanks for the feedback!